quiche/src/pmtud.rs

```diff
@@ -1,92 +1,193 @@
 #[derive(Default)]
 pub struct Pmtud {
-    /// The current path MTU estimate.
-    cur_size: usize,
+    /// The current PMTU estimate.
+    current_mtu: usize,
 
-    /// The last MTU probe size that was attempted.
-    probe: usize,
+    /// The last PMTU probe size that was attempted.
+    probe_size: usize,
 
-    /// Indicated if Path MTU probe needs to be generated.
-    next_size: bool,
+    /// Whether or not a PMTU probe needs to be generated.
+    should_probe: bool,
 
-    /// Check config for PMTU variable.
-    enable: bool,
+    /// Whether or not PMTUD is enabled.
+    enabled: bool,
 }
 
 impl Pmtud {
     /// Creates new PMTUD instance.
-    pub fn new(cur_size: usize) -> Self {
+    pub fn new(initial_mtu: usize) -> Self {
         Self {
-            cur_size,
+            current_mtu: initial_mtu,
             ..Default::default()
         }
     }
 
     /// Enables Path MTU Discovery for the connection.
     pub fn enable(&mut self, enable: bool) {
-        self.enable = enable;
+        self.enabled = enable;
     }
 
     /// Returns enable status for Path MTU Discovery for the connection.
     pub fn is_enabled(&mut self) -> bool {
-        self.enable
+        self.enabled
     }
 
     /// Specifies whether Path MTU Discovery should be performed at the next
     /// opportunity, i.e., when the next packet is sent out if possible.
     ///
-    /// Once Path MTU has been discovered, this maybe set to false.
-    pub fn should_probe(&mut self, pmtu_next: bool) {
-        self.next_size = pmtu_next;
+    /// Once Path MTU has been discovered, this may be set to false.
+    pub fn set_should_probe(&mut self, pmtu_next: bool) {
+        self.should_probe = pmtu_next;
```

Comment:
`pmtu_next` argument name seems incorrect, should be `should_probe`

Reply:
Thanks, done